src/inlineeditorui.js

```diff
@@ -150,6 +158,16 @@ export default class InlineEditorUI extends EditorUI {
 			originKeystrokeHandler: editor.keystrokes,
 			toolbar
 		} );
+
+		// Set toolbar's max-width on the initialization and update it on the editable resize,
+		// if 'shouldToolbarGroupWhenFull' in config is set to 'true'.
+		if ( !this._toolbarConfig.shouldNotGroupWhenFull ) {
```

Comment:
Since this config is now available in the editor-inline and balloon, the docs must also be updated in ckeditor5-core  https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editorconfig-EditorConfig.html#member-toolbar.

Reply:
👍